docs/development/README.md

```diff
@@ -156,3 +157,15 @@ Pocket
 ├── utility          # Implementation of the Utility module
 ├── Makefile         # [to-be-deleted] The source of targets used to develop, build and test
 ```
+
+### Linters
+
+Our CI automation runs some linters and static checks to ensure code quality. You can run them locally with:
+
+```bash
+make lint
+```
+
+You might need to install some of the linters locally, here is a list of them.
```

Comment:
Where possible/simple, can you add a make target that runs the commands to install them and update the docs here?

Reply:
@Olshansk I think it's better to keep the link to installation instructions. Snippet from `golangci-lint` docs:
> such `go install`/`go get` installation aren't guaranteed to work. We recommend using binary installation.

I'm not aware of other OS-agnostic way of installing the binary we could use in Makefile. :(

Reply:
Cool. Thanks for looking into it!